tests/swagger_generation/test_marshmallow_to_swagger.py

```diff
@@ -8,15 +8,19 @@
     :license: MIT, see LICENSE for details.
 """
 import enum
+import mock
```

Comment:
Stop it. Mock is in unittest. Stop using the old mock module.


```suggestion
from unittest import mock
```

Reply:
:shame: 😆